specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdnwebapplicationfirewall.json

```diff
@@ -888,19 +888,19 @@
     }
   },
   "parameters": {
-    "SubscriptionIdParameter": {
+    "subscriptionIdParameter": {
       "name": "subscriptionId",
       "in": "path",
       "description": "Azure Subscription ID.",
       "required": true,
       "type": "string"
     },
-    "ApiVersionParameter": {
+    "apiVersionParameter": {
       "name": "api-version",
```

Comment:
Is there reason to make these casing changes?

Reply:
We need to make these casing changes because the Python SDK generation was failing. It was failing because the casing for subscriptionIdParameter and apiVersionParameter was not consistent in the afdx.json and cdn.json files.